index.js

```diff
@@ -11,23 +11,72 @@ var asynclib = require('async')
 var jsts = require('jsts')
 var rimraf = require('rimraf')
 var overpass = require('query-overpass')
+var yargs = require('yargs')
```

Comment:
yargs is a new dependency for this library and must be added to the list of dependencies in `package.json`.

Reply:
It's cool to check in the package-lock.json file too.

Reply:
I added and updated with "npm update". LMK if that's not correct.